test/base/test_stats.py

```diff
@@ -37,4 +38,8 @@ def test_training_stats_wrapper() -> None:
         # existing fields, wrapped and not-wrapped, can be mutated
         wrapped_train_stats.loss_field = 13
         wrapped_train_stats.dummy_field = 43
-        assert wrapped_train_stats.wrapped_stats.loss_field == wrapped_train_stats.loss_field == 13
+        assert (
+            getattr(wrapped_train_stats.wrapped_stats, "loss_field")  # noqa: B009
```

Comment:
no need to use getattr, you can use `type: ignore` or `assert hasattr` to make mypy happy